fk/secretreceive.go

```diff
@@ -84,12 +91,26 @@ func secretReceive() exitCode {
 		out.Print(humanize.Pluralize(secretCount, "secret", "secrets") + ":\n\n")
 
 		for _, secret := range decryptedSecrets {
-			out.Print(formatSecretListItem(secret.decryptedContent))
-			if prompter.promptYesNo("Copy to clipboard?", "", nil) == true {
-				err := clipboard.WriteAll(secret.decryptedContent)
+			out.Print(formatSecretListItem(secret.decryptedContent, secret.filename))
+			if secret.filename != "" {
+				filenameToWrite, err := getFileNameToWrite(secret.filename)
 				if err != nil {
+					printFailed("Error prompting to save file to disk:")
 					printFailed(err.Error())
 				}
+				if prompter.promptYesNo("Save to "+filenameToWrite+"?", "", nil) == true {
```

Comment:
I would've expected this to show the *full* download path:

```
🤫─────────────────────────────
Filename: secret.txt
73599be2-2abd-11e9-a5fc-ebb0c16d6c4b
──────────────────────────────
Save to secret.txt? [y/n] 
```

Reply:
Yeah, *maybe*, I think seeing "~/Downloads" would help actually... a reminder where....

Reply:
Yeah, to remind, but also to *review* whether to allow it: I would assume `secret.txt` meant `~/secret.txt`, and would say "no".

(Also, an aside, it doesn't actually save to `~/Downloads` even if you say yes)

Reply:
Sorry - we're agreeing!